### ISSUE TITLE
RL at ICML2016/NIPS2016

### ISSUE THREAD

**Issue by sotetsuk** [sotetsuk/machine-learning-survey#159]
# [ICML2016 accepted papers](http://icml.cc/2016/?page_id=1649)
- Doubly Robust Off-policy Value Evaluation for Reinforcement Learning
  - 引用: 3
  - off-policyは実問題適用で重要だが、バイアス or バリアンスが大きい。よりunbiasでバリアンスが小さい手法を提案
- Benchmarking Deep Reinforcement Learning for Continuous Control Yan Duan University of California, Berk, Xi Chen University of California, Berkeley, Rein Houthooft Ghent University, John Schulman University of California, Berkeley, Pieter Abbeel Berkeley
  - 引用: 7
  - DL + RLはatariとかでの研究は活発だが、continuous controlではベンチマークとなるタスクがないため研究が活発でない。新しいベンチマークを提供する。
- Cumulative Prospect Theory Meets Reinforcement Learning: Prediction and Control Prashanth L.A. University of Maryland , Cheng Jie University of Maryland – College Park, Michael Fu University of Maryland – College Park, Steve Marcus University of Maryland – College Park, Csaba Szepesvari Alberta
  - 引用: 0
  - Cumulative prospect theory (CPT)というのがあって、これは意思決定におけるエビデンスのサポートに使われる。これとRLを組み合わせてrisk-sensitveなRLを考える。
- Opponent Modeling in Deep Reinforcement Learning He He , Jordan , Kevin Kwok Massachusetts Institute of Technology, Hal Daume Maryland
  - 引用: 0
  - DQNで対戦相手のpolicyを学習した
- Asynchronous Methods for Deep Reinforcement Learning Volodymyr Mnih Google DeepMind, Adria Puigdomenech Badia Google DeepMind, Mehdi Mirza , Alex Graves Google DeepMind, Timothy Lillicrap Google DeepMind, Tim Harley Google DeepMind, David , Koray Kavukcuoglu Google Deepmind
  - 引用: 18
  - We propose a conceptually simple and lightweight framework for deep reinforcement learning that uses asynchronous gradient descent for optimization of deep neural network controllers.
- Dueling Network Architectures for Deep Reinforcement Learning Ziyu Wang Google Inc., Tom Schaul Google Inc., Matteo Hessel Google Deepmind, Hado van Hasselt Google DeepMind, Marc Lanctot Google Deepmind, Nando de Freitas University of Oxford
  - 引用: 16
  - 新しいNNベースでのRL手法。アクションに対する学習が、既存RLアルゴリズムに手を加えず行える（？）
- Data-Efficient Off-Policy Policy Evaluation for Reinforcement Learning Philip Thomas CMU, Emma
  - 引用: 0
  - Doubly Robust Estimator（同じくICML2016）の発展になっている。In this paper we present a new way of predicting the performance of a reinforcement learning policy given historical data that may have been generated by a different policy.
- Model-Free Trajectory Optimization\ for Reinforcement Learning Riad Akrour TU Darmstadt, Gerhard Neumann , Hany Abdulsamad , Abbas Abdolmaleki
  - 引用: 0
  - Trajectory optimizationは局所的に線形で近似して更新することが多いが、モデルフリーで行う。

[NIPS2016](https://nips.cc/Conferences/2016/AcceptedPapers)
- Tree-Structured Reinforcement Learning for Sequential Object Localization Zequn Jie*, National Univ of Singapore; Xiaodan Liang, Sun Yat-sen University; Jiashi Feng, National University of Singapo; Xiaojie Jin, NUS; Wen Feng Lu, National Univ of Singapore; Shuicheng Yan,
  - まだpdfない
- Safe and Efficient Off-Policy Reinforcement Learning Remi Munos, Google DeepMind; Thomas Stepleton, Google DeepMind; Anna Harutyunyan, Vrije Universiteit Brussel; Marc Bellemare*, Google DeepMind
  - 引用: 0
  - off-policyにおいて、よりlow-varianceで、safe（off-policinessが高くても）で、効率的（on-policyに近いサンプルを使う）アルゴリズムを提案した。
- Contextual-MDPs for PAC Reinforcement Learning with Rich Observations Akshay Krishnamurthy*, ; Alekh Agarwal, Microsoft; John Langford, 
  - 引用: 1
  - contextual-MDPにおける新しいモデル
- Learning to Communicate with Deep Multi-Agent Reinforcement Learning Jakob Foerster*, University of Oxford; Yannis Assael, University of Oxford; Nando de Freitas, University of Oxford; Shimon Whiteson, 
  - 引用: 1
  - multi-agentがcommunication protocolを学ぶ。MNISTを使った新しい環境（タスク）で検証している。
- Hierarchical Deep Reinforcement Learning: Integrating Temporal Abstraction and Intrinsic Motivation Tejas Kulkarni, MIT; Karthik Narasimhan*, MIT; Ardavan Saeedi, MIT; Joshua Tenenbaum, 
  - 引用: 11
  - DQNで報酬を段階的に設定する必要がある問題を解いた（AtariのMontezuma's Revenge）
- Cooperative Inverse Reinforcement Learning Dylan Hadfield-Menell*, UC Berkeley; Stuart Russell, UC Berkeley; Pieter Abbeel, ; Anca Dragan, 
  - 引用: 3
  - CIRL(Cooperative Inverse Reinforcement Learning)という新しい問題を提案、このゲームにおいて人間は機械を意識せずに最適な行動を取れば良いので、使い勝手が良さそう（？） 
- Linear Feature Encoding for Reinforcement Learning Zhao Song*, Duke University; Ronald Parr, ; Xuejun Liao, Duke University; Lawrence Carin, 
  - まだpdfない
- Unifying Count-Based Exploration and Intrinsic Motivation
  - 引用: 1
  - Montezuma's Revengeも解けるようになった。
